docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
 +------------------------------------+
 | List of abbreviations              |
 +=======+============================+
-|       |                            |
+|   MM  | Multiple myeloma           |
++-------+----------------------------+
+|   RR  | Relapsed/ Refractory MM    |
```

Comment:
A more common abbreviation of relapsed/refractory multiple myeloma is "RRMM".

Reply:
I've updated the acronym to 'RRMM', thanks!